### DISCUSSION TITLE
How to change appearance of schedule view

### DISCUSSION THREAD

**Discussion by freqyfreqy** [YukiGasai/obsidian-google-calendar#87] [General]
How do I change the layout of schedule view?

I'd like my schedule to look like this: https://camo.githubusercontent.com/a91c142c1ca09584e96494ceaba361469cafea8dd979f34caacd51f388a2b0e1/68747470733a2f2f692e696d6775722e636f6d2f486b595167345a2e706e67

where the date, time, and the event name are in separate columns, which uses up screen space very cleanly and efficiently.

Instead, in my result, the date, time and event name are in separet rows which is very wasteful in screen real esate. 

I'm wondering if I could use the 'style settings' plugin to change this but I can't figure how to do this.

**Comment by freqyfreqy** [comment]
I think I know the problem. The width of the overall frame is too narrow. Is there a way to make it 'wider'?
<img width="454" alt="Screenshot 2023-03-12 at 22 11 35" src="https://user-images.githubusercontent.com/96843562/224613347-93939979-0f82-4e06-bb5b-6fd71a438dfd.png">

**Comment by freqyfreqy** [comment]
I think my issue is related to this issue here: 

https://github.com/YukiGasai/obsidian-google-calendar/issues/8

I don't know if this is a bug or its intentional.

**Comment by YukiGasai** [comment]
Hi, there is currently a pull request updating the view appearance. Generally, the view tries to find the best fit for the remaining width.
Is your problem occurring in the view, the codeblocks or both?

**Reply by freqyfreqy** [reply]
My problem occurs in my desktop(wide screen) in both Live Preview Mode as well as Reading Mode. The view in my mobile looks very similar to the desktop where the date, time, and calendar description are all on separate rows.
